iahx/templates/custom/result-citation-article.html

```diff
@@ -18,32 +18,20 @@
 
 <!-- Citation journal title -->
 {% if doc.cit_journal_title %}
-    <a href="" class="showTooltip dropdown-toggle source" data-toggle="dropdown" data-placement="top" data-original-title="{{ texts.SHOW_DETAILS }}">{{ doc.cit_journal_title }}</a>,
+    <a href="#" class="showTooltip dropdown-toggle source" data-toggle="dropdown" data-placement="top" data-original-title="{{ texts.SHOW_DETAILS }}">{{ doc.cit_journal_title }}</a>,
     <ul class="dropdown-menu">
         <li class="dropdown-header">{{ texts.SHOW_DETAILS }}</li>
         <li>
             <ul class="metricTable">
                 <li>
-                    {% if doc.cit_journal_issn_canonical %}
-                        <a href="#" class="openCitationJournalInfo" data-cit-norm-status="{{ doc.cit_normalization_status }}" data-issn-normalized="{{ doc.cit_journal_issn_normalized|join(', ') }}" data-title-normalized="{{ doc.cit_journal_title_normalized|join('#') }}" data-title="{{ doc.cit_journal_title }}" data-title-canonical="{{ doc.cit_journal_title_canonical }}" data-issn="{{doc.issn.0}}" data-issn-canonical="{{doc.cit_journal_issn_canonical|join(', ')}}">
-                            <div class="logo-statistc logo-statistc-info"></div>
-                        </a>
-                    {% else %}
-                        <a href="#" class="openCitationJournalInfo" data-title="{{ doc.cit_journal_title }}" data-issn="{{doc.issn.0}}">
-                            <div class="logo-statistc logo-statistc-info"></div>
-                        </a>
-                    {% endif %}
+                    <a href="citation-journal/{{ lang }}/{{ doc.id }}" data-toggle="modal" data-placement="top" data-target="#CitationJournalInfo">
```

Comment:
Há outro modo mais correto de passar essa rota?

Reply:
@rafaelpezzuto 

Tem mas acho que não é necessário realizarmos um ajuste de rotas nessa aplicação, pois não estamos seguindo qualquer padrão como os padrões herarquizados, prefixados, URL amigável em outros padrões que existe para URLs

É bastante estranho termos o idioma como segmento da URL se for possível e fácil acredito que a única alteração que realizaria seria: 

```
citation-journal/{{ doc.id }}/?lang=[pt, es, en]
```